Stops compilation on my linux distribution.

adampep opened the pull request:
It seems to stop compilation of all the examples as well.

adampep committed:
Stops compilation on my linux distribution.

stgeke commented:
Hmm that's odd. Anyway can you please do the proposed change for all install scripts:

./3rd_party/cvode/install
./3rd_party/gslib/install
./3rd_party/hypre/install
./3rd_party/parRSB/install
./tools/amg_setup/hypre/install
./tools/exo2nek/3rd_party/install

adampep committed:
The rest of install scripts corrected.

Everything seem to work fine except parRSB for which wget doesn't
download the file.

adampep commented:
Updated. There is some problem with downloading tar file for parRSB

stgeke commented:
That's ok - there is no parRSB release yet.
